GUI/MainModList.cs

```diff
@@ -547,9 +554,6 @@ public IEnumerable<DataGridViewRow> ConstructModList(IEnumerable<GUIMod> modules
 
                 item.Cells.AddRange(selecting, updating, name, author, installVersion, latestVersion, compat, size, desc);
 
-                selecting.ReadOnly = !mod.IsInstallable();
-                updating.ReadOnly = !mod.IsInstallable() || !mod.HasUpdate;
-
```

Comment:
These lines are still required given the exceptions thrown above.

Reply:
Also, I'm not sure why we set "ReadOnly" at all - if you try double-clicking on the cell, it ends up throwing a core exception (not in managed code), regardless of whether or not it's set as ReadOnly.

Reply:
Yeah, on further inspection I don't think we need to set ReadOnly. The entire rest of the row consists of `DataGridViewTextBoxCell` objects without ReadOnly set, so that should be fine for these cells as well.

Reply:
Uh, just re-tested without setting ReadOnly, and yes we do.  Not sure why the other cells don't enter text-edit mode (perhaps it's a column-setting?) on double-click, but these ones do and it causes issues if you start typing into them.

Reply:
Ahh, good point. The columns probably have to be editable to allow the user to change the checkboxes. I just deleted that commit, so now it's back to how you had it.

Reply:
And here we go, in Main.Designer.cs:469-534 all columns except the first two are set to read-only.